warehouse/query-core/src/main/java/datawave/query/tables/async/event/VisitorFunction.java

```diff
@@ -246,6 +241,14 @@ public ScannerChunk apply(@Nullable ScannerChunk input) {
                     if (log.isDebugEnabled()) {
```

Comment:
Given the debugging below, I think we can remove the -> newQuery  part of this debug statement.